internal/provisioner/utility_group.go

```diff
@@ -118,8 +118,13 @@ func (group utilityGroup) CreateUtilityGroup() error {
 		return errors.Wrap(err, "failed to deploy Cert Manager manifests")
 	}
 
+	// List of repos to add during helm setup
+	var helmRepos = map[string]string{
+		"jetstack": "https://charts.jetstack.io",
```

Comment:
I will add the stable as well just to be explicitly

Reply:
added stable :)